test/index.js

```diff
@@ -296,16 +296,15 @@ describe('.dest()', function() {
 
   it('outputs files where the base ends in a separator', function(done) {
```

Comment:
Huh. I'd taken "outputs files where the base ends in a separator" to mean the base of the _input_, which is true before and after this change. What do you suggest?